projects/dex-ui/src/components/Liquidity/SlippagePanel.tsx

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import React from "react";
+import styled from "styled-components";
+
+import arrowLeft from "/src/assets/images/arrow.svg";
+
+type SlippagePanelProps = {
+  slippageValue: number;
+  slippageValueChanged: (value: string) => void;
+  closeButtonClicked: () => void;
+};
+
+const SlippagePanel = ({ slippageValueChanged, closeButtonClicked, slippageValue }: SlippagePanelProps) => {
+  const buttonClickHandler = () => {
+    closeButtonClicked();
+  };
+
+  return (
+    <SlippageContainer>
+      <SlippageHeader>
+        <SlippageImageSpacer>
+          <img src={arrowLeft} alt="Back" onClick={buttonClickHandler} />
+        </SlippageImageSpacer>
+        <SlippageHeaderText>Adjust Slippage</SlippageHeaderText>
+      </SlippageHeader>
+      <SlippageBody>
+        <SlippageTopSection>
+          {/* // TODO: Can we show Slippage Tolerance as a "placeholder" (per design) */}
+          {/* // TODO: Also, we need to show a trailing % */}
+          <input type="text" defaultValue={slippageValue} onChange={(e) => slippageValueChanged(e.target.value)} />
```

Comment:
surprised this input doesn't need any styling.. haven't ran this locally. Maybe turn into into an `Input` that is styled.input`` in preparation for future design

Reply:
I'll go ahead and add the `styled.input` as you suggested.